### PR TITLE
Add torchvision.ops.FrozenBatchNorm2d to types

### PR DESCRIPTION
**Pull request by rachtibat**
torchvision.ops.FrozenBatchNorm2d is part of many SOTA models and contains the same attributes/buffers as the torch.nn.BatchNorm2d module.
Hence, it can be canonized with the existing SequentialMergeBatchNorm class.

### CODE REVIEW

**Comment by chr5tphr** [comment]
Sorry for the hold-up, I see no reason not to merge this, so I will.
